dockerfile/cuda11.1.1.dockerfile

```diff
@@ -108,5 +108,5 @@ ADD third_party third_party
 RUN make -j -C third_party cuda
 
 ADD . .
-RUN python3 -m pip install .[nvidia,torch] && \
+RUN python3 -m pip install .[nvidia,torch,ort] && \
```

Comment:
Just check. Do we need to consider AMD?

Reply:
There have no difference between CUDA and AMD for package installation.

Reply:
Got it.